RFC.doku

```diff
@@ -16,26 +18,30 @@ A deprecated feature may throw an exception before major release if the security
 
 ====== Normalization of the deprecation message ======
 
-The deprecation message must include the version of PHP in which the feature will be removed. 
+The deprecation message must include the version of PHP in which the feature will be removed.
+
+===== Defining replacements =====
+
+Active php users — taking care of php evolutions — know which features have to be used instead of the old ones: mysqli vs mysql, preg vs ereg, etc.
```

Comment:
I replaced `in replacement for` which is quite heavy by `instead of`, but it doesn't have the same strengh.